src/TimeDomainDisplayPlot.h

```diff
@@ -93,6 +93,9 @@ class TimeDomainDisplayPlot: public DisplayPlot
 
   void stemPlot(bool en);
 
+  void setPlotLineStyle(unsigned int chIdx, unsigned int style);
+  int getLineStyle(unsigned int chIdx);
```

Comment:
Shouldn't the name be consistent between a setter and a getter? (either setPlotLineStyle + getPlotLineStyle or setLineStyle + getLineStyle)